rebar.config

```diff
@@ -38,7 +38,8 @@
             {platform_define, "^2", unicode_str},
             {platform_define, "^(2[1-9])|(20\\\\.3)", filelib_find_source},
             {platform_define, "^(R|1|20)", fun_stacktrace},
-            warnings_as_errors
+            warnings_as_errors,
+            debug_info
```

Comment:
we don't want debug_info here to reduce the size of the escript. There is a specific dialyzer profile that can be used for that.

Reply:
Got it. Commit removed.

In any case, how would one know what actions to take before submitting a PR (e.g. `rebar3 as systest ct` and that sort of stuff)? Since there's no `.travis.yml` and I see this information nowhere (I'm probably not looking properly), I'd need guidance (and if we could have it in a visible place others would benefit from it 😃).

I usually do:
```erlang
rebar3 do unlock, upgrade
rebar3 do xref, dialyzer
rebar3 do eunit, ct, proper
```
(but that seems not OK for `rebar3`).

**Note**: I find a Makefile also helps, in many cases, since `make test` and `make check` should be quite intuitive.

Reply:
@paulo-ferraz-oliveira given that rebar3 uses Github Actions instead of Travis, if you want to check what the CI will do when you open a PR/push a commit you should read the [main workflow config file][main.yml] (spoiler, it runs `./rebar3 ct` :joy:).

There is also a bit more of information in the [CONTRIBUTING.md], which covers how one should request changes in the project, what is the project structure (and what are some useful modules to use), how to test the rebar3 itself and has some guidelines regarding PRs (i.e., code style, branching, etc) :)

I agree that a simple `Makefile` to wrap some basic use cases could be useful, but I guess that if the project doesn't have one it's because Fred and Tristan might have their own reasons.

[main.yml]: https://github.com/erlang/rebar3/blob/master/.github/workflows/main.yml
[CONTRIBUTING.md]: https://github.com/erlang/rebar3/blob/master/CONTRIBUTING.md

Reply:
@pablocostass, yup. Thanks. I must've skipped [CONTRIBUTING.md](https://github.com/erlang/rebar3/blob/master/CONTRIBUTING.md). In any case, it seems the tests are failing (at least to me, locally, they are).